source/puppet/5.3/configuration.md

```diff
@@ -617,6 +617,19 @@ The environment Puppet is running in.  For clients
 is used to find modules and much more.  For servers (i.e., `puppet master`)
 this provides the default environment for nodes we know nothing about.
 
+When environment is defined in the [agent] section, this refers to the
```

Comment:
This should be marked as code, i.e. `[agent]`.

Reply:
Updated